doc/manual/install-judgehost.rst

```diff
@@ -157,7 +157,8 @@ distributions may be different (send us your feedback!).
 Edit grub config to add cgroup memory and swap accounting to the boot
 options. Edit ``/etc/default/grub`` and change the default
 commandline to
-``GRUB_CMDLINE_LINUX_DEFAULT="quiet cgroup_enable=memory swapaccount=1"``
+``GRUB_CMDLINE_LINUX_DEFAULT="quiet cgroup_enable=memory swapaccount=1"``, if
+available also do the same for ``GRUB_CMDLINE_LINUX``.
```

Comment:
Do *both* really have to be changed if they exist, or does only one of these exist?

Reply:
According to https://askubuntu.com/a/780545 both can exist, GRUB_CMDLINE_LINUX will just always work but will also work when someone starts recovery mode. Maybe this user his setup was just to different so I'm fine with not merging this PR.